pygame2/pygame-2.4.0.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "pygame2",
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://files.pythonhosted.org/packages/b9/42/154038c4988d750ded799dc5d6ca4e631db93888e3d76084625b0588dfd2/pygame-2.4.0.tar.gz",
+            "sha256": "e3603e70e96ee30af1954ce57d4922a059402f368013e7138e90f1c03d185267"
+        }
+    ],
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --ignore-installed --no-deps --prefix=/app ."
```

Comment:
You should run `setup.py cython`, to recythonise the sources before installing. Upstream c sources might not match with the python api available in the runtime or app. The c api time to time introduces changes leading to build failures like https://github.com/flathub/shared-modules/issues/201 and the python version is a moving target.

https://www.pygame.org/wiki/GettingStarted#Compilation
> Some of the .c files are generated by [Cython](https://cython.org/) from .pyx files. Running "setup.py cython" will update them.